### DISCUSSION TITLE
ORDERS ARE MIXED MUCH PLEASE TAKE NOTE OF MY SUGGESTIONS.

### DISCUSSION THREAD

**Discussion by lim10soft** [Blair2004/NexoPOS#809] [Ideas]
I use translation I hope you understand my problems.

1. In the orders section, it is not written which table has an order.
2. When the table is selected, which table is selected is not written when entering the order.
3. When the table is moved to a different table. The moved table doesn't seem to be full.
![cats2](https://user-images.githubusercontent.com/102427740/161100619-d2ff9c96-673a-4cb3-bb3b-093c3d1adb32.jpg)
![cats](https://user-images.githubusercontent.com/102427740/161100662-1383fc45-6311-43e5-bb70-c39dc384d08f.jpg)

**Comment by Blair2004** [comment]
Hi, 
Thank you for your suggestion, we'll see what we can do.

**Comment by lim10soft** [comment]
thankyou
